test/unit/puppetlabs/services/ca/certificate_authority_core_test.clj

```diff
@@ -499,7 +499,7 @@
           test-app (-> (build-ring-handler settings "42.42.42")
                        (wrap-with-ssl-client-cert))]
       (testing "PUT"
-        (testing "signing a cert"
+        (testing "signing a cert with an hour ttl"
```

Comment:
Would it make sense to have an additional test, rather than amending this one? What's the desired outcome if `cert_ttl` is unset?

Reply:
In the test scenario, if no `cert_ttl` is in the payload, the code will use to what is in the `settings` for the `ca-ttl`. This condition—where no `cert_ttl` is supplied—is tested in the following test, "signing a cert with a CA that has multiple RDNs".  It didn't really seem necessary to me to test both of these explicitly.

Reply:
ahh gotcha. I totally missed that you were merging in `cert_ttl` as `:ca_ttl`, so this makes sense to me.